Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using Microsoft.Web.WebView2.Wpf;
+using Microsoft.Web.WebView2.Core;
+
+namespace Shell
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private Window _mainWindow = new Window();
+        private WebContent web = new WebContent();
+        public MainWindow()
+        {
+            InitializeComponent();
+
+
+            _mainWindow.Content = "Hello Compose!";
+            _mainWindow.Title = "Compose";
+            _mainWindow.Show();
```

Comment:
Creating and showing a window are two separate actions. I wouldn't show a window from the ctor, instead the caller should decide when/how to show it.